app/models/manageiq/providers/base_manager/manager_refresher.rb

```diff
@@ -40,7 +45,11 @@ def collect_inventory_for_targets(ems, targets)
           _log.info("Filtering inventory for #{target.class} [#{target_name}] id: [#{target.id}]...")
 
           if inventory_object_refresh?
-            inventory = builder_class_for(ems.class).build_inventory(ems, target)
+            begin
+              inventory = builder_class_for(ems.class).build_inventory(ems, target)
+            rescue NameError
+              inventory = inventory_class_for(ems.class).build(ems, target)
```

Comment:
Looks like the comment got overwritten but I think we should just `alias build_inventory build` and save the rescue here, we can remove it once we move all the providers over

Reply:
Oh just noticed the `builder_class_for` part, at first I thought the whole line was the same except for `build` / `build_inventory`